apps/air-discount-scheme/api/esbuild.json

```diff
@@ -0,0 +1,22 @@
+{
+  "platform": "node",
+  "external": [
+    "@nestjs/microservices",
+    "class-transformer",
+    "cache-manager",
+    "@nestjs/websockets/socket-module",
+    "class-validator",
+    "class-transformer",
+    "@nestjs/microservices/microservices-module",
+    "apollo-server-fastify",
+    "@elastic/elasticsearch",
+    "fastify-swagger",
+    "@nestjs/mongoose",
+    "@nestjs/typeorm",
+    "dd-trace",
+    "logform",
+    "pg-native",
+    "form-data"
```

Comment:
There was the `fsevents` we talked about, missing for mac users if they decide to run a build locally?

Reply:
right - adding `fsevents` everywhere